src/storage/src/source/kafka.rs

```diff
@@ -126,15 +126,15 @@ pub struct KafkaOffsetCommiter {
 impl SourceRender for KafkaSourceConnection {
     type Key = Option<Vec<u8>>;
     type Value = Option<Vec<u8>>;
-    type Time = Partitioned<PartitionId, MzOffset>;
+    type Time = Partitioned<RangeBound<PartitionId>, MzOffset>;
 
     const STATUS_NAMESPACE: StatusNamespace = StatusNamespace::Kafka;
 
-    fn render<G: Scope<Timestamp = Partitioned<PartitionId, MzOffset>>>(
+    fn render<G: Scope<Timestamp = Partitioned<RangeBound<PartitionId>, MzOffset>>>(
```

Comment:
A note here (or elsewhere) about why kafka uses rangebounds (awaiting refactor) would be helpful context if we both get hit by buses.

Reply:
done, thanks for the quick review!

Reply:
thank you!!!